src/crush/CrushWrapper.cc

```diff
@@ -932,7 +932,7 @@ int CrushWrapper::verify_upmap(CephContext *cct,
     return -ENOENT;
   }
   int root_bucket = 0;
-  int cursor = 0;
+  unsigned cursor = 0;
```

Comment:
could you drop this change, as it's already in #31184

Reply:
Done!